src/main/java/lab3/game/winRecord.java

```diff
@@ -0,0 +1,37 @@
+package lab3.game;
+
+/**
+ * A record that keeps tracks of both players wins over multiple games
+ *
+ * @param XWins The number of Player X's wins
+ * @param OWins The number of Player O's wins
+ */
+public record winRecord(int XWins, int OWins) {
+    private static winRecord gameScore = new winRecord(0, 0);
+
+
+    /**
+     * Retrieves the current game score
+     *
+     * @return the current winRecord
+     */
+    public static winRecord callScore() {
+        return gameScore;
+    }
```

Comment:
This feels awkward to me.  I think what you want is a local winRecord variable in your main method(s) rather than using a single class variable here.  Typically, class variables are used to store constant values, or values that are shared among all instances of the same class.